src/main/docs/guide/spring/springWithGradle.adoc

```diff
@@ -0,0 +1,23 @@
+To use micronaut-openapi with spring in gradle add this code to you `build.gradle`:
+
+.build.gradle
+[source,groovy]
+----
+dependencies {
+
+    // add to annotationProcessor and compileOnly blocks next libraries:
+
+    annotationProcessor("io.micronaut:micronaut-inject-java:$micronautCoreVersion")
+    annotationProcessor("io.micronaut.spring:micronaut-spring-annotation:$micronautSpringVersion")
+    annotationProcessor("io.micronaut.spring:micronaut-spring-web-annotation:$micronautSpringVersion")
+    annotationProcessor("io.micronaut.spring:micronaut-spring-boot-annotation:$micronautSpringVersion")
```

Comment:
```suggestion
    annotationProcessor(platform("io.micronaut.spring:micronaut-spring-bom:$micronautSpringVersion"))
    annotationProcessor("io.micronaut.spring:micronaut-spring-annotation")
    annotationProcessor("io.micronaut.spring:micronaut-spring-web-annotation")
    annotationProcessor("io.micronaut.spring:micronaut-spring-boot-annotation")
```

Reply:
I disagree with the suggestion. I didn't do it because such code could affect other dependencies in this block.